web/styles/subscriptions.css

```diff
@@ -948,8 +948,9 @@ h4.user_group_setting_subsection_title {
             justify-content: space-between;
             position: absolute;
             bottom: 0;
-            width: calc(100% - 27px);
-            padding: 9px 15px 15px;
+            /* Subtract 15px padding on either side. */
+            width: calc(100% - 30px);
```

Comment:
I'm assuming we're still calculating a width here because of the absolute positioning. It would be nice to see this reworked in Grid so such fiddliness is made unnecessary.

Reply:
Ah yeah it seems by the last commit we can remove this -- good call!